src/odemis/acq/stream/_static.py

```diff
@@ -431,16 +426,23 @@ def _onPoint(self, pos):
     def _onPolarization(self, pos):
         self._shouldUpdateImage()
 
-    def _setBackground(self, data):
-        """Called when the background is about to be changed"""
-        if data is None:
+    def _setBackground(self, bg_data):
+        """
+        Called when the background is about to be changed
+        :param bg_data: (None, DataArray or list of DataArrays) background image(s)
+        :return: (None, DataArray or list of DataArrays)
```

Comment:
Yes, that's helps to have a full docstring. I'd recommend to add _even_ more: you can add a `raises` after the `return` to indicate which types of Exceptions are raised when. Here, you'd say something like `ValueError`: the background data is not compatible with the data (ex: not the same shape or data type or polarization).